gui/src/components/Onboarding/Wrapper.tsx

```diff
@@ -0,0 +1,26 @@
+import { invoke } from "@tauri-apps/api/tauri";
+import { useState } from "react";
+
+import { useInvoke } from "../../hooks";
+import { GeneralSettings } from "../../types";
+import { OnboardingWizard } from "./Wizard";
+
+interface Props {
+  children: React.ReactNode;
+}
+
+export function OnboardingWrapper({ children }: Props) {
```

Comment:
will leave this for future work but: I'd rather see the onboarding implemented as a separate window (and configuring the app to choose on boot which one to load) than having this arbitrary wrapping of the entire app

it also means we can have some better control over e.g. window sizes: we may change the main window's size in the future, and that may not translate well to this onboarding